src/components/Common/UI/DatePicker/DatePicker.module.scss

```diff
@@ -258,6 +258,7 @@
     color: var(--g-colors-gray-1000);
     font-weight: 400;
     font-size: var(--g-typography-fontSize-small);
+    overflow-x: hidden;
```

Comment:
Small datepicker fix for showing the scroll bar that i missed when doing my input update

Reply:
Thank you!